docs/quickstart.rst

```diff
@@ -213,15 +346,24 @@ except for ``solc``:
 Install Solidity
 ~~~~~~~~~~~~~~~~
 Here's where the fun begins: you'll have to build Solidity from source, and it
-specifically needs to be the ``release_0.4.8`` branch. Here's how to do that:
+specifically needs to be the ``release_0.4.13`` branch. Here's how to do that:
```

Comment:
This is no longer correct.  It would be good to update these instructions to use the `py-solc` installation scripts.

See: https://github.com/pipermerriam/py-solc#installing-the-solc-binary